### ISSUE TITLE
calamares: selecting German does not install system fully in German, only Plasma and then its missing a lot of language packages to be manually installed

### ISSUE THREAD

**Issue by star-buck** [netrunner-rolling/2019.04-issues#19]
the other issue is that even after install in German, the system still suggests "english" language packs as well, so its even harder for people to get a well installed and localized system.

Why is that if with normal netrunner and all other systems i know one gets a well translated system when installing in a certain language?

**Comment by NuLogicSystems** [comment]
It could have something to do with keyboard layout in the installer.
I've noticed when I installed in german, that if I do not pick a german keyboard layout, etc. the system ends up half english. Also, If I pick a spanish keyboard layout then half the system would be in spanish.

I'll need to take a closer look, I only brand calamaeres, everything else comes from Manjaro.

**Comment by star-buck** [comment]
I did pick german keyboard though, but noticed it still had english us at
top.

Can you double check how Manjaro KDE does it via Calamares?

**Comment by NuLogicSystems** [comment]
Did you pick german in grub when booting the live system?
I just want to be accurate when I test on my end.

**Comment by NuLogicSystems** [comment]
Also, after install, did the MSM notifier pop-up in the tray telling you that there are language packs available?

**Comment by star-buck** [comment]
Yes, but since half of it is already German, i expected the obvious rest to
be handled during installation by Calamares too.

Is it any difference if I start the live system in english and select
German in the installer?

**Comment by NuLogicSystems** [comment]
I don't know, that's why I asked.

**Comment by star-buck** [comment]
Starting Manjaro now in english and selecting German, lets see how it ends
up.

**Comment by star-buck** [comment]
Okay, its the same.
Not the best case, but at least not anything different on our side.

**Comment by NuLogicSystems** [comment]
I'll do the same as soon as I have my morning coffee, and the latest test ISO for Manjaro-kde finishes downloading. However, I don't know why that should differ to ours, we use the same installer, and I always check their Manjaro-tools KDE profile for any changes that we would need to adopt.

**Comment by star-buck** [comment]
Its alright, no need to waste time. Now i see Cala is still 10px short, and
thinking of it it makes sense to be 800x600 insteadcof 590px. Sorry, but
could you spin one more rc6 with this? Im fine to release it then should
nothing else come up.

**Comment by NuLogicSystems** [comment]
OK, I'll do that now.

**Comment by star-buck** [comment]
hmm, even if chosen German right from the boot , somne programs do not end in German at all.
I will compare now to previous Rolling to see if it was always like that... i wonder why ubuntu and debian based versions always had the complete full translations installed, maybe lets check back with Leszek and see if it needs some module or config in Cala to download missing translations during install if internet is connected...

**Comment by NuLogicSystems** [comment]
Manjaro uses it's Manjaro settings Manager language packages kcm (module) to install translations for applications that do not have all their translations installed by default.
![Screenshot_20190422_114353](https://user-images.githubusercontent.com/10186587/56509058-f57dbe80-64f3-11e9-8ffe-ed67e468f04f.png)

Note:  MSM is disabled by default in live mode.
https://github.com/netrunner-rolling/manjaro-tools-profile/blob/master/netrunner/kde/live-overlay/etc/skel/.config/manjaro/manjaro-settings-manager.conf
